### PR TITLE
LPD-34368 Protected against empty portletResource parameter. This is …

### PR DESCRIPTION
**Pull request by brianikim**
…possible when the permission editing menu item is from a non-portlet context, for example, the product control menu which is embedded in the theme template.

@smottal ,

https://liferay.atlassian.net/browse/LPD-34368

Porting over a change that Core team initially worked on to fix https://liferay.atlassian.net/browse/LPD-34368 (Responsibility switched to USM since the code belongs to us).

Sincerely,
Brian I. Kim

### CODE REVIEW

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:test:playwright

**Comment by brianikim** [comment]
@smottal , playwright test passed locally, fairly straightforward test. We just need to click the save button in the permissions iframe. No need to update anything else in the test.

**Comment by brianikim** [comment]
ci:test:playwright

**Comment by smottal** [comment]
ci:test:relevant

**Review by smottal — APPROVED** [review]

File: modules/test/playwright/pages/portlet-configuration-web/PortletConfigurationPermissionsPage.ts [hunk 1]
```diff
@@ -22,6 +24,10 @@ export class PortletConfigurationPermissionsPage {
 	readonly successMessage: Locator;
 
 	constructor(page: Page) {
+		this.editPageLink = page.locator(
+			'.control-menu-nav-item .lfr-portal-tooltip[title="Edit"] a'
```
Comment (smottal) [inline modules/test/playwright/pages/portlet-configuration-web/PortletConfigurationPermissionsPage.ts:28]: is there another way to get this locator?

**Comment by smottal** [comment]
ci:forward

**Pull request closed without merging**